docs/source/getting_started/project_structure.rst

```diff
@@ -1,2 +1,116 @@
-Understanding the project structure
+Project structure
 ###########################################
+
+This page covers the structure of the CS Unplugged project.
+The following diagram will be helpful when reading the following sections:
+
+.. include:: ../_static/html_snippets/project_directory_tree.rst
+
+Repository directory
+=================================================
+
+The repository directory (or root directory) contains the following:
+
+- ``csunplugged/`` directory
+
+  - This directory contains the Django web system for the CS Unplugged website.
+    This includes all raw text content, images, resources, etc.
+
+- ``docs/`` directory
+
+  - This directory contains the documentation for the repository (which includes
+    the file you are reading now).
+
+- ``subtitles/`` directory
+
```

Comment:
Consider removing the word 'directory' from each of the three directory paths above. It's quite jarring to read 'csunplugged/' etc. (in its orange box) straight into 'directory' (in normal text).
It is called a directory at the start of each description anyway, and it becomes consistent with 'config' below (line 45).